src/app/chart/data.ts

```diff
@@ -0,0 +1,14 @@
+export var single = [
```

Comment:
I wonder what's data.ts for?  Is it used somewhere? I didn't find anywhere that uses it.

Reply:
data.ts was initially used for when I was trying out different chart libraries but it got pushed by accident.
It has been deleted now.